### PR TITLE
Freshen up VS Code doc a bit

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
# PR Summary

Fixes https://github.com/MicrosoftDocs/PowerShell-Docs/issues/5576
Freshens up a lot of the content in the Using-vscode.md doc.

cc @SydneyhSmith @rjmholt 

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://docs.microsoft.com/powershell/scripting/community/contributing/overview) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/components/vscode/using-vscode.md [hunk 1]
```diff
@@ -23,47 +28,48 @@ macOS, and Linux, see the following links:
...
 For traditional Windows PowerShell workloads, see [Installing Windows PowerShell][install-winps].
 
-## Editing with VSCode
+> [!NOTE]
+> Visual Studio Code is not the same as [Visual Studio](https://visualstudio.microsoft.com/).
 
-1. Install VSCode. For more information, see the overview
-   [Setting up Visual Studio Code](https://code.visualstudio.com/Docs/setup/setup-overview).
+> [!IMPORTANT]
+> The [Windows PowerShell ISE][ise] is also still available for Windows, however, it is no longer in
```
Comment (rjmholt) [inline reference/docs-conceptual/components/vscode/using-vscode.md]: Maybe worth mentioning here that the ISE doesn't support PS 6 or 7

File: reference/docs-conceptual/components/vscode/using-vscode.md [hunk 2]
```diff
@@ -6,13 +6,18 @@ ms.date: 11/07/2019
...
 # Using Visual Studio Code for PowerShell Development
 
-In addition to the [PowerShell ISE][ise], PowerShell is also well-supported in Visual Studio Code
-(VSCode). The ISE isn't supported with PowerShell Core, but VSCode is supported for PowerShell Core
-on all platforms: Windows, macOS, and Linux.
+Visual Studio Code is a cross-platform (Windows, macOS, and Linux) script editor by Microsoft.
+With Visual Studio Code and
+[the PowerShell extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell)
+you are provided a rich and interactive script editing experience allowing you to write scripts with
+ease and confidence.
```
Comment (rjmholt) [inline reference/docs-conceptual/components/vscode/using-vscode.md]: I'd remove the second person here

```suggestion
[Visual Studio Code](https://code.visualstudio.com/) is a cross-platform (Windows, macOS, and Linux) script editor by Microsoft.
Together with the
[the PowerShell extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell),
it provides a rich interactive script editing experience,
making it easier to write reliable PowerShell scripts.
```

File: reference/docs-conceptual/components/vscode/using-vscode.md [hunk 3]
```diff
@@ -76,16 +82,16 @@ receiving an error such as:
 Language server startup failed.
 ```
 
-To manually approve PowerShell Editor Services and the PowerShell extension for VSCode, open a
-PowerShell prompt and run the following command:
+To manually approve PowerShell Editor Services and the PowerShell extension for Visual Studio Code,
+open a PowerShell prompt and run the following command:
```
Comment (rjmholt) [inline reference/docs-conceptual/components/vscode/using-vscode.md]: Might be worth mentioning in this section that constrained language mode isn't currently supported

Comment (TylerLeonhardt) [reply reference/docs-conceptual/components/vscode/using-vscode.md]: added

File: reference/docs-conceptual/components/vscode/using-vscode.md (hunk 2 above)
Comment (TylerLeonhardt) [reply reference/docs-conceptual/components/vscode/using-vscode.md]: Changed

File: reference/docs-conceptual/components/vscode/using-vscode.md (hunk 1 above)
Comment (TylerLeonhardt) [reply reference/docs-conceptual/components/vscode/using-vscode.md]: Added

**Pull request merged**